lms/static/js/components/header/views/header.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * A generic header view class.
+ */
+;(function (define) {
+    'use strict';
+    define(['backbone', 'text!templates/components/header/header.underscore'],
+           function (Backbone, headerTemplate) {
+               var HeaderView = Backbone.View.extend({
+                   initialize: function (options) {
+                       this.template = _.template(headerTemplate);
+                   },
+
+                   render: function () {
+                       var json = this.model.toJSON();
```

Comment:
I'm wondering whether this view should update if the model updates? It is a bit strange to use a model and then not handle updates. If we think that the model will never change, it might be simpler to just pass options to the view rather than a model.